app/models/course/discussion/topic.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 class Course::Discussion::Topic < ActiveRecord::Base
   actable
+  class_attribute :global_topic_models
+  self.global_topic_models ||= []
```

Comment:
Ooo. I just found a problem here.

If we just stash the reference to the class, when the class is changed in development mode, the class reference won't be updated. The old code will remain if we use this.

The acts_as_discussion_topic should stash the class *name*, then use `constantize`.